lib/widgets/content_art.dart

```diff
@@ -430,14 +430,12 @@ class _SongScopedStorageArtSourceLoader extends _ArtSourceLoader {
           size: Size.square(size) * MediaQuery.of(state.context).devicePixelRatio,
           signal: _signal!,
         );
-      } on SweyerPluginException catch (ex, stack) {
-        if (ex != SweyerPluginException.io) {
-          FirebaseCrashlytics.instance.recordError(
-            ex,
-            stack,
-            reason: 'in _SongScopedStorageArtSourceLoader.load',
-          );
-        }
+      } on SweyerPluginIoException catch (ex, stack) {
+        FirebaseCrashlytics.instance.recordError(
```

Comment:
This is now doing exactly the opposite of what is supposed to do.

We specifically to record don't want IO exceptions here, because they are caused by non-existing art (which is not our fault the system provided to us) and by canceling out the loading.

Reply:
Ah, sorry, you are right.